CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 The following is a curated list of changes in the Enact project, newest changes on the top.
 
+## [unreleased]
```

Comment:
In enact repo, dev dependency update does not need to write change log.
Please delete this comment.

Reply:
OK